src/app/pages/private/collection/collection.page.html

```diff
@@ -48,10 +49,11 @@
     }
 
     @if(isDataLoading()){
-    <div class="tw-flex tw-justify-center tw-items-center tw-w-full tw-h-full">
+    <div class="tw-flex tw-justify-center tw-items-center tw-w-full">
       <ion-spinner name="bubbles" color="medium" />
     </div>
-    } @else if (collectionInfoError() !== undefined) {
+    <app-cards-list-skeleton/>
```

Comment:
it will be also nice to add skeleton for chip view (cards can be presented as images or chips). Feel free to create for it separate ticket or implement it in current

Reply:
will crreate a separate ticket